src/rgw/rgw_file.cc

```diff
@@ -1213,10 +1213,19 @@ namespace rgw {
     return dar;
   } /* RGWFileHandle::decode_attrs */
 
-  bool RGWFileHandle::reclaim() {
+  bool RGWFileHandle::reclaim(const cohort::lru::ObjectFactory* newobj_fac) {
     lsubdout(fs->get_context(), rgw, 17)
       << __func__ << " " << *this
       << dendl;
+    auto factory = dynamic_cast<const RGWFileHandle::Factory*>(newobj_fac);
+    if (factory == nullptr) {
+      return false;
```

Comment:
Does this mean we can only reclaim handles from the RGWFileHandle::Factory?  Are no other handles reclaimable?

Reply:
the func reclaim is override by RGWFileHandle in rgw_file.cc，so it must be RGWFileHandle::Factory here。other handlers should realize  recliam func and use their factory .

Reply:
Ah, missed that, thanks.